csharp/Platform.Collections/Arrays/ArrayPool[T].cs

```diff
@@ -44,12 +70,31 @@ public Disposable<T[]> Resize(Disposable<T[]> source, long size)
             return destination;
         }
 
+        /// <summary>
+        /// <para>Clears the pool.</para>
+        /// <para>Очищает пул.</para>
+        /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public virtual void Clear() => _pool.Clear();
 
+        /// <summary>
+        /// <para>Retrieves an array with the specified size from the array pool.</para>
+        /// <para>Извлекает из пула массивов массив с указанным размером.</para>
```

Comment:
Just `an array` from `the pool`. It is obvious that the pool contains arrays. The comment will be shorter this way.